Workiva/w-mobile-kit
# MOBILE-2304: Create panel component in Mobile Kit

Description
---
For the native viewer components, we need an expandable panel component that you can drag up with it snapping to specific heights.

What Was Changed
---
Added components:
- WPanelVC, WPagingPanelVC
  - WPagingPanelVC is a subclass of WPanelVC that has built in support for adding pages managed by a UIPageViewController
  - Constrained by size classes, so that when the view the panel exists in has a width larger than the cap (which can be changed from the default value), it will switch to a side panel from the vertical extendable panel
- WFAButton
  - A simple floating action button that you can set properties on including colors and icons, and use constraints

Testing
---
* Ensure unit tests pass and have proper code coverage
* Play with panel in example app, especially on Plus iPhone and iPads that support multitasking and play with orientation/screen changes
* Ensure floating action button feels natural
* Ensure PanelExample.swift is a broad example showcasing all features of WPanel
* Ensure everything in UX Spec is achievable with basic implementation

---

Please Review: @Workiva/mobile
